sdk/eventhubs/azure-messaging-eventhubs/test/ut/eventhubs_test_base.hpp

```diff
@@ -12,7 +12,7 @@
 enum class AuthType
 {
   Key,
-  ConnectionString,
+  //  ConnectionString,
```

Comment:
Why the comment, why not remove it? Along with some other commented code in this PR. If we drop it, I'd just remove it. For history, or if we ever want this back, there is git history.